src/Components/Blazor/DevServer/src/Microsoft.AspNetCore.Blazor.DevServer.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <AssemblyName>blazor-devserver</AssemblyName>
+    <PackageId>Microsoft.AspNetCore.Blazor.DevServer</PackageId>
+    <IsShippingPackage>true</IsShippingPackage>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
+    <StartupObject>Microsoft.AspNetCore.Blazor.DevServer.Program</StartupObject>
```

Comment:
Why is this required?

Reply:
Because there's more than one `static class Program` with a `static void Main`. We could rename things but specifying this here is equally good.